package.yaml

```diff
@@ -125,6 +126,8 @@ tests:
       - juvix
       - tasty
       - tasty-hunit
+      - tasty-quickcheck
+      - QuickCheck
```

Comment:
I will need to write custom generator for test data generation, so I need to add QuickCheck too.